src/main/java/com/conveyal/r5/analyst/GridBounds.java

```diff
@@ -0,0 +1,8 @@
+package com.conveyal.r5.analyst;
+
+/**
+ * We have a lot of classes which have zoom, west, north, width, and height attributes. We would like to be able to pass
+ * them into functions without breaking out the values of these.
+ */
+public class GridBounds {
```

Comment:
It seems like this is supposed to be an interface allowing the same methods to be called on several classes, but it doesn't define any abstract methods or fields.